src/particle/converters/__init__.py

```diff
@@ -10,12 +10,16 @@
 
 from .evtgen import EvtGen2PDGNameMap, EvtGenName2PDGIDBiMap, PDG2EvtGenNameMap
 from .geant import Geant2PDGIDBiMap
+from .lhcb import LHCb2PDGNameMap, LHCbName2PDGIDBiMap, PDG2LHCbNameMap
```

Comment:
This import I would not make explicit, meaning by default. No need for 99.9% of people using the package after all.

We do need to document the availability of the maps, but that's a totally different matter.